src/backend/optimizer/plan/aqumv.c

```diff
@@ -422,6 +421,12 @@ answer_query_using_materialized_views(PlannerInfo *root,
 		{
 			root->parse = mvQuery;
 			root->processed_tlist = subroot->processed_tlist;
+			root->agginfos = subroot->agginfos;
+			root->aggtransinfos =  subroot->aggtransinfos;
```

Comment:
hasNonCombine need assign ?

Reply:
Good suggestion, I will have a look.

Reply:
Fixed, we need to correct them, as well as another three related to agg.
Though, some of them may not be used due to IVM and AQUMV process, we keep them right.
Thanks for reviewing.